resources/connector/connector.py

```diff
@@ -657,6 +657,9 @@ async def run_command(self, command):
     if command.startswith("/test ") or command.startswith("/run "):
       self.coder.io.running_shell_command = True
       self.coder.io.tool_output("Running " + command.split(" ", 1)[1])
+    elif command.startswith("/tokens"):
+      self.coder.io.running_shell_command = True
```

Comment:
Not ideal to have it as a command output here, but given Aider’s `cmd_tokens` implementation, using `send_log_message` would require copying too much code into the connector. So let’s stick with this solution.